contributors/devel/api_changes.md

```diff
@@ -347,21 +347,25 @@ before starting "all the rest".
 
 ### Edit types.go
 
```

Comment:
Somewhere in this doc:

If you add a new APi version, add that new version to `KUBE_AVAILABLE_GROUP_VERSIONS` in hack/lib/init.sh